ICRCs/ICRC-7/ICRC-7.md

```diff
@@ -6,7 +6,13 @@
 
 # ICRC-7: Base Non-Fungible Token (NFT) Standard
 
-The ICRC-7 is a standard for the base implementaion of Non-Fungible Tokens (NFTs) on the [Internet Computer](https://internetcomputer.org).
+// Open issues
+
+// - specify error codes
+
+// - Royalties: there were suggestions to remove the royalties elements entirely
```

Comment:
Considering the fact that royalties can be defined in multiple ways and in any token. I would opt to keep it out of the ICRC-7 base standard. This would also resolve the issues around accounts in metadata and metadata methods.

Reply:
Yes, we will remove the royalties-related specifications from the draft and move it to another ICRC standard as discussed in yesterday's WG meeting.